### PR TITLE
Adapt instructions so they also work on Hortense

### PR DESCRIPTION
**Pull request by stevenvdb**
This PR adapts the scripts and the README to get cryoSPARC running on Hortense.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -138,15 +201,56 @@ clusters. This is automated with the `connect_clusters.sh` script:
...
 Now you can visit the address obtained after starting the cryoSPARC master
 in a browser and login with the credentials specified when creating a cryoSPARC
-user. From there you can follow the tutorial provided here:
+user. There are a few different ways to do this.
+
+The first option is to launch a browser on the node where you launched the
+cryoSPARC master process. At the KU Leuven cluster, this can be done for
+instance by opening a NoMachine desktop and simply using the browser installed
+there. On Hortense, you can visit the [OnDemand platform](https://tier1.hpc.ugent.be)
+and launch a Cluster Desktop there. This will allow you to open a browser on a
```
Comment (tatianawoller) [inline README.md:223]: people can use Cluster Desktop or Bioimage Analysis Desktop or Neurodesk (depending if they need specific software for image analysis after using cryoSPARC

File: README.md [hunk 2]
```diff
@@ -108,16 +157,30 @@ Now the cryoSPARC master process can be started by running
 
 ```cryosparcm start```
```
Comment (tatianawoller) [inline README.md:163]: On tier-1, I need to use ``` .\cryosparcm start ```

**Review by tatianawoller — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (stevenvdb) [reply README.md:223]: Done in e0a13e56c8c92cba048fbec331d395624d17079f

File: README.md (hunk 2 above)
Comment (stevenvdb) [reply README.md:163]: `.\cryosparcm start` will only work if you are inside the installation directory of the cryosparc master. If you first run `source set_environment.sh`, this installation directory is prepended to `$PATH` and you can just use `cryosparc start`. This has been clarified in 20c2c2b0661171bad84f3c441b473db0b56fff68

**Pull request merged**